ppr-ui/src/composables/mhrInformation/useTransportPermits.ts

```diff
@@ -147,6 +160,21 @@ export const useTransportPermits = () => {
     return payloadData
   }
 
+  // Pre-fill Transport Permit for Amendment
+  const prefillTransportPermit = () => {
+
+    const homeLocationInfo: MhrRegistrationHomeLocationIF  = getMhrRegistrationLocation.value
+    const ownLand = getMhrInformation.value.permitLandStatusConfirmation
+
+    // Set original Transport Permit for future comparison with Amendment filing
+    setMhrOriginalTransportPermit({ key: 'newLocation', value: homeLocationInfo })
```

Comment:
@cameron-eyds added setter for Original Transport Permit

Reply:
Beauty, nice work!